### DISCUSSION TITLE
preset for wgs data

### DISCUSSION THREAD

**Discussion by QLZhouBio** [milaboratory/mixcr#1835] [Q&A]
Hello, I have some wgs data constructed using IDT single strand lib prep kit and would like to check the potential TCR diversity. Might I know which preset should I selected?
And also, might I know is there any DNA data I could download to test the analysis? 
Thanks.

**Comment by mizraelson** [comment]
Hi,
The command bellow should work:

```
mixcr analyze exome-seq \
    --species hsa \
      input_R1.fastq.gz \
      input_R2.fastq.gz \
      result
```

We do not have any test WGS datasets, but you can use [SRA explorer](https://sra-explorer.info/) to find some public data.

**Reply by QLZhouBio** [reply]
thanks a lot. I will try it:)
